openmetadata-spec/src/main/resources/json/schema/entity/applications/configuration/external/automator/lineagePropagationAction.json

```diff
@@ -19,9 +19,57 @@
       "$ref": "#/definitions/lineagePropagationActionType",
       "default": "LineagePropagationAction"
     },
+    "propagateChildrenDescription": {
+      "title": "Propagate Column Description",
+      "description": "Propagate description through lineage",
+      "type": "boolean",
+      "default": true
+    },
+    "propagateChildrenTags": {
+      "title": "Propagate Column Tags",
+      "description": "Propagate tags through lineage",
+      "type": "boolean",
+      "default": true
+    },
+    "propagateChildrenGlossaryTerms": {
```

Comment:
should we call this as propagateColumnLevel, 
IMO the granularity should be 
1. Documentation
2. Glossary Terms
3. Tags
4. Tier
5. Owner
6. Domain?
7. TestCases?
Then have a simple option to propagate to columns
one can select one or all the above options and propagate or not to columns

Reply:
I like it. I'll make the changes. I can add domain as well, test cases I'll iterate